app/src/main/res/layout/main_fragment.xml

```diff
@@ -8,78 +8,76 @@
     tools:context=".UI.MainFragment">
 
 
-        <AutoCompleteTextView
-            android:id="@+id/rcpSearch"
+        <LinearLayout
+            android:id="@+id/linearLayout"
             android:layout_width="0dp"
             android:layout_height="wrap_content"
-
-            android:layout_marginTop="41dp"
-            android:hint="Select a recipe"
-
+            android:layout_marginStart="10dp"
+            android:layout_marginLeft="10dp"
+            android:layout_marginEnd="10dp"
+            android:layout_marginRight="10dp"
+            android:gravity="center"
+            android:orientation="horizontal"
             app:layout_constraintEnd_toEndOf="parent"
             app:layout_constraintStart_toStartOf="parent"
-            app:layout_constraintTop_toTopOf="parent"
-            android:text="Select a Recipe"
```

Comment:
Emptied the text so all we have is a hint of this. Just a hint allows the user to jump right in and start typing instead of needing to delete your text first.